packages/tools/client-logger/app-insights-logger/package.json

```diff
@@ -0,0 +1,133 @@
+		"start:test-app:client": "webpack serve --config webpack.config.js",
+		"start:tinylicious": "tinylicious",
+		"test": "npm run test:jest",
+		"test:coverage": "npm run test:jest:coverage",
+		"test:jest": "jest --detectOpenHandles",
+		"test:jest:coverage": "jest --detectOpenHandles --coverage --ci --reporters=default --reporters=jest-junite",
+		"tsc": "tsc",
+		"webpack": "webpack --env production",
+		"webpack:dev": "webpack --env development"
+	},
+	"dependencies": {
+		"@fluentui/react": "^8.96.1",
+		"@fluentui/react-hooks": "^8.6.12",
+		"@fluentui/react-icons-mdl2": "^1.3.22",
+		"@fluentui/react-tabs": "^9.1.6",
+		"@fluid-experimental/react-inputs": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluid-tools/client-debugger": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/common-definitions": "^0.20.1",
+		"@fluidframework/container-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/container-loader": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/core-interfaces": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/counter": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/datastore": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/driver-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/map": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/protocol-definitions": "^1.1.0",
+		"@fluidframework/sequence": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/shared-object-base": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/test-driver-definitions": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@microsoft/applicationinsights-web": "^2.8.11",
+		"re-resizable": "^6.9.9",
+		"react": "^17.0.1",
+		"react-dom": "^17.0.1",
+		"react-json-view": "^1.21.3",
+		"scheduler": "^0.20.0"
+	},
+	"devDependencies": {
+		"@fluid-example/collaborative-textarea": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluid-tools/webpack-fluid-loader": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/build-common": "^1.1.0",
+		"@fluidframework/build-tools": "^0.13.1",
+		"@fluidframework/eslint-config-fluid": "^2.0.0",
+		"@fluidframework/fluid-static": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/test-runtime-utils": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@fluidframework/tinylicious-client": ">=2.0.0-internal.4.1.0 <2.0.0-internal.5.0.0",
+		"@microsoft/api-extractor": "^7.34.4",
+		"@testing-library/dom": "^8.2.0",
+		"@testing-library/jest-dom": "^5.16.5",
+		"@testing-library/react": "^12.0.0",
+		"@testing-library/user-event": "^14.4.3",
+		"@types/jest": "22.2.3",
+		"@types/react": "^17.0.44",
+		"@types/react-dom": "^17.0.18",
+		"@types/testing-library__jest-dom": "^5.14.5",
+		"concurrently": "^6.2.0",
```

Comment:
The version of `concurrently` doesn't match what the rest of the client-release group is using, we'll want to make them match. I think there's some CI check in place or coming soon that would point this out, but if we haven't run into it, we should double check versions of common deps against the ones in other packages.

Reply:
updated in commit 70aa9fd